dockerfiles/ubuntu-xenial

```diff
@@ -17,7 +17,9 @@
 # NOTE: These are intended to be built using the arguments as
 # described in ../build.sh. See that script for more details.
 
-FROM ubuntu:xenial
+FROM BASEARCHREPO/ubuntu:xenial
+
+CROSS_BUILD_COPY bin/qemu-ARCH-static /usr/bin/
```

Comment:
Again just create a new `ubuntu-xenial-ppc64le` and make your changes, no need for `sed` trickery.

Reply:
yep, understood.